docs/ocis/backup.md

```diff
@@ -0,0 +1,201 @@
+---
+title: "Backup Considerations"
+date: 2024-05-07T10:31:00+01:00
+weight: 30
+geekdocRepo: https://github.com/owncloud/ocis
+geekdocEditPath: edit/master/docs/ocis
+geekdocFilePath: backup.md
+---
+
+This small guide aims to shed some light on the internal ocis data structure. You can refer to it when you are trying to optimize your backups or if you just curious how ocis stores its data.
+
+## Ocis data structure
+
+Ocis stores its data in folder that can be configured via the envvar `OCIS_BASE_DATA_PATH`.
+
+The default value for the `OCIS_BASE_DATA_PATH` variable is `$HOME/.ocis` (or `/etc/dev/ocis` when using the docker container.
```

Comment:
It's `/var/lib/ocis/` for the container builds. And the config dir defaults to `/etc/ocis`  in the container images.